main/monmon.rb

```diff
@@ -11,6 +11,9 @@
 balance = Hash.new(0)
 not_supported = Hash.new(0)
 SUPPORTED_CURRENCIES = %w[BYN USD RUR EUR].freeze
+rates = {BYN: 1, USD: 2.5846, RUR: 0.0336, EUR: 2.7943}
+main_currency = :BYN
```

Comment:
that's perfect! now let's think deeper: what will happen if I change `main_currency` to `USD`? how to make the code work?